plugins/woocommerce/includes/admin/class-wc-admin-importers.php

```diff
@@ -1,4 +1,4 @@
-<?php
+<?php //phpcs:ignore Generic.PHP.RequireStrictTypes.MissingDeclaration
```

Comment:
This is an ancient file and I think it's best to keep it as it is for backward compatibility.